src/payment/PaymentForm.jsx

```diff
@@ -34,6 +39,8 @@ export class PaymentFormComponent extends React.Component {
       throw new SubmissionError(errors);
     }
 
+    sdnCheck(firstName, lastName, city, country);
```

Comment:
Ideally this call would be made in the sagas to isolate side effects (like making requests) outside the components to keep them more pure.